src/main/java/com/stories/service/StoriesServiceImpl.java

```diff
@@ -126,6 +128,64 @@ public StoryDomain updateStory(StoryDomain storyDomain, String id) throws Except
 			throw new EntityNotFoundException(mensaggeDinamicValidation[0], mensaggeDinamicValidation[1]);
 		}
 	}
+	
+	@Override
+	public TasksDomain updateTask(TasksDomain task, String id, String _id) throws Exception {
+		if(storiesRepository.existsById(id)) {//Validating that the story exists
+			task.set_id(_id);
+			TaskModel taskModel = storiesCustomRepository.getTaskById(id, _id).getUniqueMappedResult();
+			//Validating that the task exists
```

Comment:
Please remove the comments of this class.

Reply:
Solved